_includes/pbc/all/install-features/202410.0/enable-dynamic-multistore.md

```diff
@@ -0,0 +1,56 @@
+This document describes how to enable [Dynamic Multistore](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/dynamic-multistore-feature-overview.html) on the latest codebase.
+
+## Enable Dynamic Multistore
+
+{% info_block warningBox "Project version" %}
+If your project version is below 202307.0, you need to update to the latest codebase first, see [Install Dynamic Multistore](/docs/pbc/all/dynamic-multistore/202410.0/base-shop/install-dynamic-multistore.html) page.
+{% endinfo_block %}
+
+### Here are the steps that needs to be performed to enable Dynamic Multistore.
+
+{% info_block warningBox "Staging environment" %}
+     Make sure that **all** steps above are performed (and fully tested) on staging before applying it on production setup, to avoid unexpected downtime and data loss.
+{% endinfo_block %}
+
+
+1. Make sure that your custom Backoffice, MerchantPortal, Console Commands, Gateway, BackendAPI code do not use `StoreFacade::getCurrentStore()` as well as `StoreClient::getCurrentStore()` methods, they are no longer available in any other application except GlueStorefront and Storefront. 
```

Comment:
It would be good to move https://docs.spryker.com/docs/pbc/all/dynamic-multistore/202410.0/base-shop/dynamic-multistore-feature-overview.html#how-dynamic-multistore-affects-a-project here and leave reference at https://docs.spryker.com/docs/pbc/all/dynamic-multistore/202410.0/base-shop/dynamic-multistore-feature-overview.html to docs/pbc/all/dynamic-multistore/202410.0/base-shop/difference-between-modes.md